### DISCUSSION TITLE
ftp://ftp.ncbi.nlm.nih.gov/refseq/H_sapiens/mRNA_Prot/human.rna.fna.gz 链接失效

### DISCUSSION THREAD

**Discussion by ZHONGHONGXIN** [ShixiangWang/self-study#107] [Q&A] [answered]
师兄您好，我现在遇到一个难题。
           human.rna.fna.gz这个文件如果ncbi的链接失效的话，我还可以去什么地方寻找？下载最新的文件跑程序会报错:IndexError: list index out of range. 而且源代码1000多行我不知从哪里入手去改。所以急需原始的human.rna.fna.gz 文件。
           
           如果师兄有空的话，麻烦帮我看看这个。
           十分感谢您。

                                                                                                                                      学生
                                                                                                                                     钟宏新

**Comment by ShixiangWang** [comment, marked as answer]
用浏览器登录 https://ftp.ncbi.nlm.nih.gov/refseq/H_sapiens/mRNA_Prot/

**Reply by ZHONGHONGXIN** [reply]
谢谢您。
